examples/safety_and_security/retail_agent/README.md

```
@@ -370,14 +389,69 @@ Workflow Result: ['The customer has been sent an email with the product informat
+
+```yaml
+workflow_functions:
+  retail_tools__get_product_info:
+    reviews:
+      - review
+```
+
+Each product review is sent to the rail on its own, separate from the rest of the product. A masked review is written back in place (the product object is preserved); a review that must be blocked replaces the whole response with the refusal message.
+
+> **Validation:** Field paths in the mapping form are validated when the function is registered, against the function's input or output schema. A path that names a field that does not exist — or that resolves to a non-string field — raises an error immediately, rather than silently guarding nothing. A valid path whose data happens to be empty at runtime (for example, a product with no reviews) is simply a no-op.
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)